apps/trivium/src/static_deque/static_bit_deque.rs

```diff
@@ -0,0 +1,136 @@
+//! This module implements the StaticBitDeque struct: a deque of bytes. The idea
```

Comment:
StaticBitDeque or StaticByteDeque as it seems to store bytes ?

Really a detail

Reply:
This is much better, I don't like StaticBitDeque at all